atom/browser/web_dialog_helper.cc

```diff
@@ -56,10 +108,13 @@ void WebDialogHelper::RunFileChooser(content::WebContents* web_contents,
     }
 
     std::vector<base::FilePath> paths;
+    auto browser_context = AtomBrowserMainParts::Get()->browser_context();
```

Comment:
Actually all `content::BrowserContext` created in Electron should be an `AtomBrowserContext`, it is safe to call `static_cast`.